### PR TITLE
Add documentation to edit-comments.js

### PR DESCRIPTION
**Pull request by Xyfi**
Props to: @manuelaugustin, @diedexx, @thulshof and @xyfi.
Fixes https://github.com/Yoast/wordpress-develop-mirror/issues/30

### CODE REVIEW

File: src/wp-admin/js/edit-comments.js [hunk 1]
```diff
@@ -631,6 +831,15 @@ commentReply = {
...
 
+	/**
+	 * Close the comment edit or reply form.
+	 *
+	 * @since 3.5.0
+	 *
+	 * @memberOf commentReply
+	 *
+	 * @returns {void}
+	 */
```
Comment (diedexx) [inline src/wp-admin/js/edit-comments.js]: Please start inline comments in this function with an uppercase letter and end them with full stops

Comment (diedexx) [reply src/wp-admin/js/edit-comments.js]: Could you also change `replyrow is not showing?` so it's not a question

File: src/wp-admin/js/edit-comments.js [hunk 2]
```diff
@@ -786,6 +1017,18 @@ commentReply = {
...
+	 * Shows the new or updated comment or reply.
+	 *
+	 * When the passed xml is a string, it will be considered an error.
+	 *
+	 * @since 3.5.0
+	 *
+	 * @memberOf commentReply
+	 *
+	 * @param {Object} xml     Ajax response object.
+	 * @returns {boolean|void} Returns either nothing or false if an error occurred.
```
Comment (diedexx) [inline src/wp-admin/js/edit-comments.js]: Please add a newline between param and returns

File: src/wp-admin/js/edit-comments.js [hunk 3]
```diff
@@ -849,6 +1092,16 @@ commentReply = {
...
 	},
 
+	/**
+	 * Shows an error for the failed comment update or reply.
+	 *
+	 * @since 3.5.0
+	 *
+	 * @memberOf commentReply
+	 *
+	 * @param {string} r The Ajax response.
```
Comment (diedexx) [inline src/wp-admin/js/edit-comments.js]: Please add newline

File: src/wp-admin/js/edit-comments.js [hunk 4]
```diff
@@ -865,6 +1118,16 @@ commentReply = {
...
 	},
 
+	/**
+	 * Opens the add comments form in the comments metabox on the post edit page.
+	 *
+	 * @since 3.5.0
+	 *
+	 * @memberOf commentReply
+	 *
+	 * @param post_id The post id.
```
Comment (diedexx) [inline src/wp-admin/js/edit-comments.js]: Please add newline

**Comment by atimmer** [comment]
Committed in https://core.trac.wordpress.org/changeset/45558.

**Pull request closed without merging**